front50-sql-postgres/front50-sql-postgres.gradle

```diff
@@ -1,3 +1,3 @@
 dependencies {
-  runtimeOnly "org.postgresql:postgresql:42.2.8"
+  runtimeOnly "org.postgresql:postgresql:42.2.13" // to fix CVE-2020-13692
```

Comment:
There seem to be enough places that depend on `org.postgresql:postgresql` that I think it makes sense to move the dependency to [kork](https://github.com/spinnaker/kork/blob/master/spinnaker-dependencies/spinnaker-dependencies.gradle).  For me [this search](https://github.com/search?q=org%3Aspinnaker+org.postgresql%3Apostgresql&type=code) returns clouddriver, fiat, front50, and orca.